microsoft-edge/extensions/publish/marketing-guidelines.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Marketing guidelines for Microsoft Edge Add-ons
+description: Marketing guidelines for publishing an extension, theme, or web app, such as using badges, at Microsoft Edge Add-ons.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 08/13/2025
+---
+# Marketing guidelines for Microsoft Edge Add-ons
+<!-- todo: is this content copied (then modified) from a webpage? which? -->
```

Comment:
is this content copied (then modified) from a webpage? which?

that could affect eg moving sections around; affects how heavily to alter this article

@G-u-r-j-e-e-t

Reply:
upstream doc worked yesterday, 404 now

your PR [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/3550/files/df2baa3be93b5de944d77bf2df56e97e2a24d5c9#r2281501767) contains /en-in/ link:

https://learn.microsoft.com/en-in/windows/apps/publish/app-marketing-guidelines - 404
https://learn.microsoft.com/en-us/windows/apps/publish/app-marketing-guidelines - 404
https://learn.microsoft.com/windows/apps/publish/app-marketing-guidelines - 404

@G-u-r-j-e-e-t  fyi